### PR TITLE
MBS-8303 remove mockwebserver spam in tests

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

File: subprojects/common/test-okhttp/src/main/kotlin/com/avito/test/http/MockDispatcher.kt [hunk 1]
```diff
@@ -16,15 +25,15 @@ class MockDispatcher(
 
         capturers.find { it.requestMatcher.invoke(request) }?.run {
             capture(request)
-            println("request captured: $request")
+            logger.info("request captured: $request")
```
Comment (sergioboi) [inline subprojects/common/test-okhttp/src/main/kotlin/com/avito/test/http/MockDispatcher.kt:28]: debug?

Comment (dsvoronin) [reply subprojects/common/test-okhttp/src/main/kotlin/com/avito/test/http/MockDispatcher.kt:28]: there is no debug in java.util.Logger that is used here

Comment (eugene-krivobokov) [reply subprojects/common/test-okhttp/src/main/kotlin/com/avito/test/http/MockDispatcher.kt:28]: what about `com.avito.logger.Logger`? Is it for such kotlin libraries?

**Review by eugene-krivobokov — APPROVED** [review]

**Review by sergioboi — APPROVED** [review]

File: subprojects/common/test-okhttp/src/main/kotlin/com/avito/test/http/MockDispatcher.kt (hunk 1 above)
Comment (dsvoronin) [reply subprojects/common/test-okhttp/src/main/kotlin/com/avito/test/http/MockDispatcher.kt:28]: Mockwebserver uses java.util under the hood, and it seems overcomplicated to redirect it to our logger

**Pull request merged**